### PR TITLE
Feature/updating pipelines

### PR DESCRIPTION
**Pull request by gpoleze**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/api-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: API - Deploy ecrc-api
+on:
+  workflow_dispatch:
+    inputs:
+      source_image_tag:
+        required: true
+        description: What is the tag that is going to be promoted?
```
Comment (BrendanBeachBC) [inline .github/workflows/api-deploy.yml:7]: Which tag is this referring to, is this the GitHub tag?

File: .github/workflows/frontend-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+name: Frontend - Deploy ecrc-api
```
Comment (BrendanBeachBC) [inline .github/workflows/frontend-deploy.yml:1]:
```suggestion
name: Frontend - Deploy ecrc-frontend
```
Small copy paste error

File: .github/workflows/frontend-deploy.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+on:
+  workflow_dispatch:
+    inputs:
+      source_image_tag:
+        required: true
+        description: What is the tag that is going to be promoted?
```
Comment (BrendanBeachBC) [inline .github/workflows/frontend-deploy.yml:7]: Same question as above, is this the GitHub tag?

**Review by BrendanBeachBC — APPROVED** [review]
Looks good, a couple of minor things

File: .github/workflows/frontend-deploy.yml (hunk 3 above)
Comment (gpoleze) [reply .github/workflows/frontend-deploy.yml]: Changed. It now says "image tag."

**Pull request merged**